test/iron-overlay-behavior.html

```diff
@@ -631,7 +637,88 @@
           assert.equal(overlay.getAttribute('aria-hidden'), 'true', 'overlay has aria-hidden="true"');
         });
 
-      })
+      });
+
+      suite('restore focus', function() {
+
+        var overlay;
+        setup(function () {
+          overlay = fixture('autofocus');
+        });
+
+        test('_deepestActiveElement handles document.body', function () {
+          document.body.focus();
+          assert.equal(overlay._deepestActiveElement, document.body);
+        });
+
+        test('_deepestActiveElement handles light dom', function () {
+          var focusable = document.createElement('input');
+          document.body.appendChild(focusable);
+          focusable.focus();
+          assert.equal(overlay._deepestActiveElement, focusable, 'input is handled');
+          document.body.removeChild(focusable);
+        });
+
+        test('_deepestActiveElement handles shadow dom', function (done) {
+          var focusable = fixture('focusables').$.focusable0;
+          // Wait for rendering before setting focus
+          setTimeout(function () {
```

Comment:
this timeout and the ones at line 676 and 692 are needed for tests on IE10.
I've tried to use `Polymer.dom.flush()`, `TestHelpers.flushAsynchronousOperations()` and `TestHelpers.forceXIfStamp(fixture)`, but no luck with any. 
This is the only way I could find to have the tests passing until we :skull: IE10 from our tests.